src/app/InteractionModelEngine.cpp

```diff
@@ -361,6 +361,40 @@ void InteractionModelEngine::ReleaseClusterInfoList(ClusterInfo *& aClusterInfo)
     aClusterInfo               = nullptr;
 }
 
+void InteractionModelEngine::ReleaseClearClusterInfoList(ClusterInfo *& aClusterInfo)
+{
+    ClusterInfo * updateList = nullptr;
+    ClusterInfo * runner     = aClusterInfo;
+    if (aClusterInfo == nullptr)
+    {
+        return;
+    }
```

Comment:
It looks like this null check is not needed.

Reply:
yep, agree, thanks